### PR TITLE
updated with token card styling

### PR DESCRIPTION
**Pull request by eupston**
here's the initial styling for displaying the tokens and collection title

### CODE REVIEW

File: nft-demo/src/App.js [hunk 1]
```diff
@@ -20,23 +24,35 @@ function NFTCollection(props) {
...
+        {tokenInfo?.map(({tokenId, tokenDetails, owner}) => {
+          tokenId = tokenId.toString()
+          tokenDetails = JSON.parse(tokenDetails.toString())
+          owner = owner.toString()
+          return (
+          <div className="flip-card">
+            <div className="flip-card-inner">
+              <div className="flip-card-front">
+                <div className={cardHovered ? 'shine' : ''}>
+                  <img
```
Comment (KarishmaBothara) [inline nft-demo/src/App.js:38]: img elements must have an alt prop, either with meaningful text, or an empty string for decorative images
Can we add this property
alt={'xxx'}

File: nft-demo/src/App.js [hunk 2]
```diff
@@ -57,7 +73,14 @@ function App() {
   }
 
   return (
+    <div className="App">
+      <div>
+        <h1 className="neonText"> NFT DEMO</h1>
+        <h3 className="neonText">Collection: {collectionId}</h3>
+        <h5 className="neonText"></h5>
```
Comment (KarishmaBothara) [inline nft-demo/src/App.js]: can we remove h5?

**Comment by KarishmaBothara** [comment]
> here's the initial styling for displaying the tokens and collection title

styling looks cool 👍

File: nft-demo/src/App.js (hunk 1 above)
Comment (eupston) [reply nft-demo/src/App.js:38]: yup sure thing

**Review by KarishmaBothara — APPROVED** [review]

**Pull request merged**